prow/cluster/monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -9,7 +9,8 @@ stringData:
       metrics_path: /metrics
       static_configs:
         - targets:
-            - "104.197.27.114"
+            - "104.197.27.114:9090"  # boskos resource metrics
+            - "fillmein-boskos-http-metrics:9090"  # boskos http metrics
```

Comment:
We should probably wait to add this until we have a valid value to avoid failing scrapes.